extensions/wikia/AdEngine/AdEngine2ExitstitialHooks.class.php

```diff
@@ -115,7 +115,7 @@ static private function getExitstitialUrlsWhiteList() {
 		}
 
 		$wikiDomains = WikiFactory::getDomains( $wgCityId );
-		if ( $wikiDomains !== false ) {
+		if ( is_array($wikiDomains) ) {
```

Comment:
It fixes PHP warning and shouldn't break anything with existential. But I'm worried `WikiFactory::getDomains( $wgCityId );` returned empty list or string.